requirements/requirements_lightning.txt

```diff
@@ -1,6 +1,6 @@
 hydra-core>1.3,<=1.3.2
 omegaconf<=2.3
-pytorch-lightning>=2.0,<=2.0.7
+pytorch-lightning==2.2.1
```

Comment:
should be >= 2.2.1

Reply:
Fixed in the latest commit. @ericharper let me know if you have any other review comments. The CI passed fully and we should be good to merge after the code review.